lighthouse-core/gather/gather-runner.js

```diff
@@ -165,6 +166,20 @@ class GatherRunner {
     } else if (mainRecord.hasErrorStatusCode()) {
       errorDef = {...LHError.errors.ERRORED_DOCUMENT_REQUEST};
       errorDef.message += ` Status code: ${mainRecord.statusCode}.`;
+    } else if (!mainRecord.finished) {
```

Comment:
AFAIK, an insecure security state will always have `.finished` set to false on the network records. I wanted to avoid calling these Security commands if not needed.